introduction/overview.md

```diff
@@ -3,22 +3,22 @@
+Valtimo does not strive to support a broad tech stack: it is an opinionated Business Process Automation implementation - often following trends in the market.  
 
-The core team uses the following guide lines: 
+The core team uses the following guidelines: 
 
-1. Use before make. Try to prevent inventing wheels. 
-2. Use micro services with a thin integration layer, making them replaceable in the future. 
+1. Use before make. Do not reinvent the wheel.
+2. Use microservices with a thin integration layer, making them replaceable in the future. 
 3. Keep things simple. 
-4. Use upcoming but proven technology. 
+4. Use upcoming, but proven technology. 
 
 ## When to use Valtimo? 
 
-There is no universal, broadly excepted definition of Business Process Automation categories. A possible categorization in types is: 
+There is no universal, broadly accepted definition of Business Process Automation categories. A possible categorization of types is: 
 
-1.	**Workflow**. Being the simplest form of BPA, often largely consisting sequential steps in a predefined process. Examples are the approval flow for holidays-requests or onboarding new employees. Using Valtimo for these types of processes is not recommended. Using a point solution is often the better option. 
+1.	**Workflow**. Being the simplest form of Business Process Automation, often largely consisting of sequential steps in a predefined process. Examples are the approval flow for holidays requests or onboarding new employees. Using Valtimo for these types of processes is not recommended. Using a point solution is often the better option. 
 
-2.	**Service Orchestration**. In large scale micro service landscapes, communication between services can be a challenge. To step away from point to point connections, communication can be handled with a choreography-pattern with a message bus solution like Kafka, AWS SQS or RabbitMQ. The alternative option is to use an Orchestration pattern using a BPMN-engine. These types of processes typically are seen enterprise environments, focussing on eliminating manual work. Typically high volume processes with no or minimal manual work. Using Valtimo for service Orchestration is not recommended. Using a scalable BPMN-engine is advised. 
+2.	**Service Orchestration**. In large scale microservice landscapes, communication between services can be a challenge. To step away from point-to-point connections, communication can be handled with a choreography pattern with a message bus solution like Kafka, AWS SQS or RabbitMQ. The alternative option is to use an orchestration pattern using a BPMN engine. These types of processes are typically seen in enterprise environments, focussing on eliminating manual work. Typically high volume processes with no or minimal manual work. Using Valtimo for Service Orchestration is not recommended. Using a scalable BPMN engine is advised. 
```

Comment:
I think `Service Orchestration should` not be capitalized at all in the second to last sentence.